packages/headless/src/controllers/breadcrumb-manager/headless-breadcrumb-manager.ts

```diff
@@ -192,13 +222,61 @@ export function buildBreadcrumbManager(
     };
   };
 
+  const getAutomaticFacetBreadcrumbs = (): AutomaticFacetBreadcrumb[] => {
+    const set = getState().automaticFacetSet?.set;
+    if (set) {
+      return Object.values(set).map((slice) =>
+        buildAutomaticFacetBreadcrumb(slice.response)
+      );
+    }
+    return [];
```

Comment:
Could be shortened to (just like above):
```suggestion
    const set = getState().automaticFacetSet?.set ?? {};
    return Object.values(set).map((slice) =>
      buildAutomaticFacetBreadcrumb(slice.response)
    );
```